packages/interfaces/src/electronAPI/updater.ts

```diff
@@ -0,0 +1,50 @@
+export interface UpdateInfo {
+  /**
+   * The version.
+   */
+  readonly version: string;
+  /**
+   * The release name.
+   */
+  releaseName?: string | null;
+  /**
+   * The release notes. List if `updater.fullChangelog` is set to `true`, `string` otherwise.
+   */
+  releaseNotes?: string | null;
```

Comment:
```suggestion
  releaseNotes?: string | string[] | null;
```
Comment suggests it can be a list